account_invoice_shipping_address/stock.py

```diff
@@ -19,16 +19,23 @@
 #
 ##############################################################################
 
-from openerp.osv import orm
+from openerp import models
 
 
-class StockPicking(orm.Model):
+class StockPicking(models.Model):
     _inherit = "stock.picking"
 
     def _get_invoice_vals(self, cr, uid, key, inv_type,
-                          journal_id, picking, context=None):
+                          journal_id, move, context=None):
```

Comment:
Why not changing also this to new API with `@api.model`? And was the signature incorrect previously?